scheduling-manager/src/test/java/com/amazonaws/blox/scheduling/manager/ManagerHandlerTest.java

```diff
@@ -37,9 +39,25 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class ManagerHandlerTest {
-  public static final String CLUSTER_ARN = "arn:::::cluster1";
-  public static final String FIRST_ENVIRONMENT_ARN = "arn:::::environment1";
-  public static final String SECOND_ENVIRONMENT_ARN = "arn:::::environment2";
+  public static final String ACCOUNT_ID = "12345678012";
+  public static final String CLUSTER_NAME = "cluster1";
+  public static final String FIRST_ENVIRONMENT_NAME = "environment1";
+  public static final String SECOND_ENVIRONMENT_NAME = "environment2";
+
+  private static final Cluster CLUSTER =
+      Cluster.builder().accountId(ACCOUNT_ID).clusterName(CLUSTER_NAME).build();
+  private final EnvironmentId firstEnvironmentId =
```

Comment:
could be also a static for environment ID since we don't intent to be different for different test case.

Reply:
Good catch. Fixed